playbooks/charon-devel.yml

```diff
@@ -0,0 +1,30 @@
+- hosts: devel
+  become: true
+  vars:
+    katello_repositories_version: nightly
+    foreman_installer_scenario: katello-devel
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--katello-devel-github-username {{ katello_devel_github_username }}"
+      - "--katello-devel-enable-ostree=true"
+    foreman_installer_additional_packages:
+      - foreman-installer-katello-devel
+    foreman_installer_options:
```

Comment:
Should this also be on internal so users can override other settings easily in boxes.d?

Reply:
Fair point.